### PR TITLE
Update checkout mutations

### PR DESCRIPTION
**Pull request by mateuszgrzyb**
I want to merge this change because...

Extend checkoutPaymentCreate mutation [SALEOR-4229](https://app.clickup.com/t/wku4v1)
Update CheckoutComplete mutation [SALEOR-4232](https://app.clickup.com/t/wkug8f)
Update payment flow [SALEOR-4233](https://app.clickup.com/t/wkukb1)

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4232 Update CheckoutComplete mutation](https://app.clickup.com/t/2549495/SALEOR-4232)

**Comment by patrys** [comment]
Task linked: [SALEOR-4229 Extend checkoutPaymentCreate mutation](https://app.clickup.com/t/2549495/SALEOR-4229)

**Comment by patrys** [comment]
Task linked: [SALEOR-4233 Update payment flow](https://app.clickup.com/t/2549495/SALEOR-4233)

File: saleor/graphql/payment/mutations.py [hunk 1]
```diff
@@ -194,6 +256,9 @@ def perform_mutation(cls, _root, info, checkout_id=None, token=None, **data):
             customer_ip_address=get_client_ip(info.context),
             checkout=checkout,
             return_url=data.get("return_url"),
+            store_payment_method=data.get("store_payment_method")
```
Comment (korycins) [inline saleor/graphql/payment/mutations.py:259]: Not sure if this `OR` is necessary here. `input['store_payment_method']` has default value to `StorePaymentMethod.NONE`, so there is no way to have `None` here (AFAIK)

**Comment by korycins** [comment]
@mateuszgrzyb  It looks good. Let's solve the last comment and we can merge it 👍 

Most of the review has been done in this PR - https://github.com/mirumee/saleor/pull/7956

**Review by korycins — APPROVED** [review]

**Pull request merged**